packages/code-du-travail-nlp/api/search.py

```diff
@@ -0,0 +1,12 @@
+from flask import request
+from flask import jsonify
+from flask_cors import CORS, cross_origin
+
+def add_search(app, ss):
+    @app.route('/api/search', methods=['GET'])
+    @cross_origin()
+    def search():
+        query = request.args.get('q')
```

Comment:
is it possible to support a size parameter like the node js api ?

Reply:
Done in https://gitlab.factory.social.gouv.fr/SocialGouv/code-du-travail-numerique/commit/0620facc302cdba268c1a725d75ba56836d7e0bf